style.css

```diff
@@ -174,11 +174,11 @@ input[type="submit"],
 textarea:focus,
 .button,
 .gallery img {
-	-webkit-transition: all 0.1s ease-in-out;
-	-moz-transition:    all 0.1s ease-in-out;
-	-ms-transition:     all 0.1s ease-in-out;
-	-o-transition:      all 0.1s ease-in-out;
-	transition:         all 0.1s ease-in-out;
+	-webkit-transition: all .1s ease-in-out;
```

Comment:
And I'm giving a reason why this particular suggestion should be changed back to include a leading `0`, so there is more chance of the whole PR being accepted. There's no reason to deviate away from documented code standards for something this trivial.